.cirrus.yml

```diff
@@ -213,11 +196,33 @@ qa_ruling_kotlin_task:
     ITS_PROJECT: "ruling"
     GIT_SUB_MODULE: "its/sources"
   <<: *LINUX_6_CPU_12G_JAVA_17
-  <<: *KOTLIN_ITS_TEMPLATE
+  <<: *GRADLE_ITS_TEMPLATE
+
+qa_ruling_kotlin_compiler_task:
+  <<: *QA_TASK_FILTER
+  env:
+    KOTLIN_COMPILER_ENABLED: "true"
```

Comment:
Minor, I found the flag name a little bit confusing - how about renaming it to something like `KOTLIN_COMPILER_IT_ENABLED`?